docs/ocs-endpoint-v2.md

```diff
@@ -168,3 +168,12 @@ In order to delete a notification, you can send a DELETE request against `/ocs/v
 In order to delete all notifications, you can send a DELETE request against `/ocs/v2.php/apps/notifications/api/v2/notifications`
 
 **Note:** This endpoint was added for Nextcloud 14, so check for the `delete-all` capability first.
+
+
+## Check existance of notifications for a user
+
+In order to check whether a set of notification ids (max. 200 items per request) still exist for a user,
+a client can send a POST request against `/ocs/v2.php/apps/notifications/api/v2/notifications/exists` with
+the integer list provided as `ids` field on the POST body.
+
+**Note:** This endpoint was added for Nextcloud 27, so check for the `exists` capability first.
```

Comment:
26.0.1 in this case

Reply:
No one checks stable branch docs, but yeah might want to adjust the master docs.
Will do in a follow up when replacing #1461

Reply:
Added in #1461